src/services/helpers.ts

```diff
@@ -19,3 +19,15 @@ export const getArticleColor = (article: Article): string => {
       return COLORS.lunesArtikelDer
   }
 }
+
+/* Randomize array using Durstenfeld shuffle algorithm with O(n) complexity */
+export const shuffleArray = <T>(array: T[]): T[] => {
+  const shuffledArray = array.slice()
+  for (let i = shuffledArray.length - 1; i > 0; i--) {
```

Comment:
Better to use build in array functions like `forEach`

Reply:
It iterates backwards. Even through it should not make a difference, this is the original algorithm.

There is a shorter version of this:

```
function shuffleArray(array) {
    for (let i = array.length - 1; i > 0; i--) {
        const j = Math.floor(Math.random() * (i + 1));
        [array[i], array[j]] = [array[j], array[i]];
    }
}
```
https://stackoverflow.com/a/12646864/1763110

Reply:
@hepperle00 looks good, there is one additional ;